lib/abci/handlers/checkTxHandlerFactory.js

```diff
@@ -18,7 +23,21 @@ function checkTxHandlerFactory(decodeStateTransition) {
    * @returns {Promise<abci.ResponseCheckTx>}
    */
   async function checkTxHandler(request) {
-    await decodeStateTransition(request.tx);
+    const { tx } = request;
+
+    if (!tx) {
```

Comment:
I suppose it is impossible. You can't submit an empty transaction to Tendermint.